lib/awful/completion.lua

```diff
@@ -90,7 +90,7 @@ function completion.shell(command, cur_pos, ncomp, shell)
         i = i + 1
     end
 
-    if cword_index == 1 and not string.find(words[cword_index], "/") then
+    if cword_index == 1 then
```

Comment:
CC @Asido Any thoughts on this and https://github.com/awesomeWM/awesome/commit/3986409e70a3ec5602512c0d9a148ba0a57cb015 ?